### ISSUE TITLE
Problem in Jenkins

### ISSUE THREAD

**Issue by vadym-kukhtin** [zalando-incubator/solution-center-feedback#77]
We in CMS Team have problem with load sc-feedback as a dependency in package.json.

When we are running it inside our CI Jenkins pipeline, during npm installation phase we see error:

14:36:06 > solution-center-feedback@1.2.0 postinstall /opt/app/node_modules/solution-center-feedback
14:36:06 > typings install
14:36:06 typings: not found

**Comment by gregdeane** [comment]
@cjhowald It might be related to the fact that `postinstall` expects a global installation of `typings` so we should probably create an npm script `"typings": "npm run typings"`.

I'm guessing that will work.

**Comment by vadym-kukhtin** [comment]
@gregdeane You are completely right!

**Comment by gregdeane** [comment]
@vadym-kukhtin It may take a bit of time to research and implement the right solution so in the meantime, could you add `typings` as a dev dependency to your main project (the one being handled by Jenkins)?

This should fix the issue you are having until we can implement the correct solution.

Thx!

**Comment by vadym-kukhtin** [comment]
@gregdeane There is the problem here - we have typings as a dependency, but problem is still there.

**Comment by gregdeane** [comment]
@vadym-kukhtin I'm able to get this to work locally when I add `typings` to my `dependencies` object in package.json. I think this means it should also work on Jenkins.

I tried two scenarios:

**Scenario 1: Using the current solution-center-feedback version**
My test repo package.json looks like this:
```json
"dependencies": {
  "solution-center-feedback": "^1.2.0",
  "typings": "^1.3.2"
}
```

**Scenario 2: I also updated solution-center-feedback to add a separate npm script for `typings`**
My test repo package.json looks like this:
```json
"dependencies": {
  "solution-center-feedback": "git://github.com/zalando-incubator/solution-center-feedback.git#bugfix/77-typings-issues-in-jenkins",
  "typings": "^1.3.2"
}
```

Both scenario's above work as expected. Typings installs without problem.

I don't know what to make of your Jenkins build but if I'm not mistaken, the tests above illustrate that if `typings` is installed as a dependency in your project, the call to `typings install` in solution-center-feedback should work.
